### PR TITLE
Add title and jobs offerts

### PR DESCRIPTION
**Pull request by maretamara**
Dodałam tytuły oraz kolejne dwie oferty pracy 

* [x] @beatadelura 
* [x] @MagdaMalinowska 
* [ ] @kruszczynski 
* [ ] @AdamMankowski 
* [ ] @rpietraszko 
* [ ] @korzonek

### CODE REVIEW

**Comment by MagdaMalinowska** [comment]
Looks good for me :)

File: source/partials/_job-offers.html.slim [hunk 1]
```diff
@@ -1,10 +1,13 @@
 .wrapper.small
+  h1.m-content-heading.large.color-martinique-dark.font-alega-bold
+    '
+      Software Development
```
Comment (beatadelura) [inline source/partials/_job-offers.html.slim]: In this case, you don't need to use two additional lines. It's enough to use one:
`h1.m-content-heading.large.color-martinique-dark.font-alega-bold Software Development` or if it's too long: 
```
h1.m-content-heading.large.color-martinique-dark.font-alega-bold
  |  Software Development
```

File: source/partials/_job-offers.html.slim [hunk 2]
```diff
@@ -34,9 +37,34 @@
...
+      requirements_tags: '#elixir #phoenix',
       bcg_class: 'bcg-breaker-bay-green',
       requirements_class: 'deep-sea-green',
       btn_class: 'viridian-green-wave',
       temporary_styles: 'temporary-styles',
       job_position_link: '/join-our-crew/elixir'}
+
+  h1.m-content-heading.large.color-martinique-dark.font-alega-bold
+    '
+      Customer Development
```
Comment (beatadelura) [inline source/partials/_job-offers.html.slim]: same here :)

File: source/stylesheets/modules/_m-job-offers.scss [hunk 3]
```diff
@@ -21,7 +25,8 @@
   .m-job-offers {
     &.m-grid-collection {
       justify-content: space-between;
-      margin-top: 80px;
+      margin-top: 40px;
+      margin-bottom: 40px;
```
Comment (beatadelura) [inline source/stylesheets/modules/_m-job-offers.scss]: Why don't use shorthand for margin?

**Comment by beatadelura** [comment]
Looks good with small correction 👍

**Comment by beatadelura** [comment]
bitmoji

**Pull request merged**